scripts/fires/wfigs_data.py

```diff
@@ -0,0 +1,227 @@
+"""Script to import and clean WFIGS data.
```

Comment:
please don't forget the copyright header

Reply:
Done